templates/frontend/pages/indexJournal.tpl

```diff
@@ -66,7 +68,8 @@
 
 	{* Latest issue *}
 	{if $issue}
-		<div id="homepageIssue" class="current_issue">
+		<div class="current_issue" role="contentinfo">
```

Comment:
The `role="contentinfo" should be removed from this div as well as the announcements and about divs above. This landmark role should only be used once for the page footer.

It probably makes sense to make each of these a `<section>`.